files/en-us/web/javascript/reference/global_objects/string/replace/index.html

```diff
@@ -49,13 +49,15 @@ <h3 id="Parameters">Parameters</h3>
   <dd>The {{jsxref("String")}} that replaces the substring specified by the specified
     <code><var>regexp</var></code> or <code><var>substr</var></code> parameter. A number
     of special replacement patterns are supported; see the "<a
-      href="#Specifying_a_string_as_a_parameter">Specifying a string as a parameter</a>"
-    section below.</dd>
+      href="#specifying_a_string_as_a_parameter">Specifying a string as a parameter</a>"
+    section below.
+    <p>If <code><var>newSubstr</var></code> is the empty string, then the substring given by <code><var>substr</var></code>, or the matches for <code><var>regexp</var></code>, are removed (rather then being replaced).</p>
```

Comment:
"an empty string" rather than "the empty string"?
```suggestion
    <p>If <code><var>newSubstr</var></code> is an empty string, then the substring given by <code><var>substr</var></code>, or the matches for <code><var>regexp</var></code>, are removed (rather then being replaced).</p>
```